miden/tests/integration/stdlib/crypto/fri/mod.rs

```diff
@@ -30,14 +30,14 @@ fn fri_fold4_ext2_remainder32() {
     let depth = trace_len_e + blowup_exp;
     let domain_size = 1 << depth;
 
-    let (advice_provider, tape, position_eval, alphas, commitments, remainder, num_queries) =
+    let (advice_provider, stack, position_eval, alphas, commitments, remainder, num_queries) =
```

Comment:
I think `stack` here should also be changed to `advice_stack`.

Reply:
Fixed on #785